copilot/action.yaml

```diff
@@ -6,3 +6,7 @@ author: 'Ariga'
 runs:
   using: node20
   main: index.js
+inputs:
+  gen-test:
+    description: Generate atlas tests for the schema changes.
+    required: false
```

Comment:
Please add new inputs in this file instead:

https://github.com/ariga/atlas-action/blob/2facfabc29e284af0f9b1bbee5bb0649936e14e5/atlas-actions.yml#L84-L86

Then run `make manifest`